x-pack/test/api_integration/apis/monitoring/elasticsearch/ccr.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from 'expect.js';
```

Comment:
Nice that we have some integration tests 👍 This is especially useful to know in case the API changes (I assume we would get an error then). Does this also detect in case a field is missing that we need from the result?

Reply:
Yup!